frontend/src/apiMocks/handlers.ts

```diff
@@ -3,554 +3,578 @@ import { CustomAliasData, RandomAliasData } from "../hooks/api/aliases";
 import { UnverifiedPhone, VerifiedPhone } from "../hooks/api/realPhone";
 import { RelayNumber } from "../hooks/api/relayNumber";
 import { ProfileData } from "../hooks/api/profile";
-import {
-  mockIds,
-  mockedDomainaddresses,
-  mockedProfiles,
-  mockedRelayaddresses,
-  mockedRuntimeData,
-  mockedUsers,
-  mockedRealphones,
-  mockedRelaynumbers,
-  mockedInboundContacts,
-} from "./mockData";
-
-export function getHandlers(
-  defaultMockId: null | (typeof mockIds)[number] = null,
-): HttpHandler[] {
-  const handlers: HttpHandler[] = [];
-
-  const getMockId = (req: Request): (typeof mockIds)[number] | null => {
-    const authHeader = req.headers.get("Authorization");
-    if (typeof authHeader !== "string") {
-      return defaultMockId;
-    }
-
-    const token = authHeader.split(" ")[1];
-    return mockIds.find((mockId) => mockId === token) ?? defaultMockId;
-  };
-
-  const addGetHandler: (...args: Parameters<typeof http.get>) => void = (
-    path,
-    resolver,
-  ) => {
-    handlers.push(http.get(path, resolver));
-    handlers.push(http.get(`http://127.0.0.1:8000${path}`, resolver));
-  };
-  const addPatchHandler: (...args: Parameters<typeof http.patch>) => void = (
-    path,
-    resolver,
-  ) => {
-    handlers.push(http.patch(path, resolver));
-    handlers.push(http.patch(`http://127.0.0.1:8000${path}`, resolver));
-  };
-  const addPostHandler: (...args: Parameters<typeof http.post>) => void = (
-    path,
-    resolver,
-  ) => {
-    handlers.push(http.post(path, resolver));
-    handlers.push(http.post(`http://127.0.0.1:8000${path}`, resolver));
-  };
-  const addDeleteHandler: (...args: Parameters<typeof http.delete>) => void = (
-    path,
-    resolver,
-  ) => {
-    handlers.push(http.delete(path, resolver));
-    handlers.push(http.delete(`http://127.0.0.1:8000${path}`, resolver));
-  };
-
-  addGetHandler("/accounts/logout", (_info) => {
-    return HttpResponse.text(null, { status: 200 });
-  });
-
-  addGetHandler("/api/v1/runtime_data", (_info) => {
-    return HttpResponse.json(mockedRuntimeData, { status: 200 });
-  });
 
-  addGetHandler("/api/v1/users/", (info) => {
-    const mockId = getMockId(info.request);
-    if (mockId === null) {
-      return HttpResponse.text(null, { status: 401 });
-    }
-
-    return HttpResponse.json([mockedUsers[mockId]], { status: 200 });
-  });
+let handlers: HttpHandler[] = [];
+
+if (process.env.NEXT_PUBLIC_MOCK_API === "true") {
+  const {
+    mockIds,
+    mockedDomainaddresses,
+    mockedProfiles,
+    mockedRelayaddresses,
+    mockedRuntimeData,
+    mockedUsers,
+    mockedRealphones,
+    mockedRelaynumbers,
+    mockedInboundContacts,
+    // necessary to prevent mocks leaking into prod
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
```

Comment:
suggestion (non-blocking): move this docstring up to the top of this `if` block?